BricksAndHearts.UnitTests/ServiceTests/Api/ApiServiceTests.cs

```diff
@@ -67,12 +65,12 @@ public void TurnResponseBodyToModel_WhenCalledWithNonEmptyString_ReturnsNonEmpty
         }
         var results = postcodeApiResponseViewModel.ListOfResults[0];
         results.Address.Should().NotBeNull("Address should not be null");
-        if (results.Address == null)
-        {
-            return;
-        }
-        results.Address.Keys.ToList().Should().Contain("streetName");
-        results.Address["streetName"].Should().Be("Adam & Eve Street");
+        results.LatLon.Should().NotBeNull("LatLon should not be null");
+
+        results.Address?.Keys.ToList().Should().Contain("streetName");
+        results.Address?["streetName"].Should().Be("Adam & Eve Street");
+        results.LatLon?["lat"].Should().Be("52.2046");
+        results.LatLon?["lon"].Should().Be("0.13244");
```

Comment:
Avoid `?` in assertions unless you really only want to run the assertion if the thing is non-null. Here I'm pretty sure we want it to fail if `results.LatLon` is null
```suggestion
        results.LatLon["lat"].Should().Be("52.2046");
        results.LatLon["lon"].Should().Be("0.13244");
```

Reply:
Yeah the reason I added the question mark is because without it, Rider will complain and I can't pass the github tests.

Reply:
Because of the possible null reference

Reply:
In that case use `!`.

* `Obj?.Method()` means if `Obj` is null then do nothing
* `Obj!.Method()` means try and call it anyway, and fail with a null pointer exception if `Obj` is null

Generally you shouldn't use `!` much in regular code, but in tests it's fine.